usr/share/66/service/sndiod

```diff
@@ -6,9 +6,8 @@
 @options = ( log env )
 
 [start]
-@build = custom
-@shebang = "/bin/sh"
-@execute = ( exec sndiod ${OPTS} -d )
+@execute = ( execl-cmdline -s { sndiod ${cmd_args} -d } )
 
 [environment]
-OPTS=
+# Value is the default as a placeholder - parser cannot use empty values.
```

Comment:
if i have an empty var, i deleted it. adding can make the admin. to learn, how is mustbe, imho. in runit admin has to do that too..

Reply:
I absolutely agree, that is also my approach!
That is a really difficult one because the options are frequently needed and are included in the void-documentation :(
https://docs.voidlinux.org/config/media/sndio.html?highlight=sndio#sndio

Reply:
Ι think I will leave it as-is for now. Let's see if other services have the same issue with existing documentation and decide then :)

Reply:
Ok.